theme/ItaliaTheme/Views/_common.scss

```diff
@@ -113,16 +113,21 @@ picture.volto-image.responsive img.full-width {
     margin-bottom: 1rem !important;
     font-size: 1.4rem;
   }
-
-  .card-title {
-    margin: 0 !important;
-    margin-bottom: 1rem !important;
-    line-height: 1em;
-
-    &:not(.big-heading) a {
-      color: $primary;
-      font-size: 1.125rem;
-      line-height: 1.625rem;
+  .card {
+    .card-body {
+      h5.card-title.big-heading {
+        font-size: 24px;
```

Comment:
Mettere la dimensione in `rem` invece che `px`

Reply:
@pnicolli done